frontend/tests/playwright/test-fixtures.ts

```diff
@@ -1,14 +1,65 @@
+/**
+ * Playwright test fixtures are very powerful and can be used to create a custom
+ * test environment.
+ *
+ * Fixtures can be isolated to a test or a worker depending on the scope defined.
+ *
+ * In addition, fixtures can be lazily loaded, which means that the fixture will only be created where it is used. If a test scoped fixture is not used in a test, it will not be created. If a worker scoped fixture is not used in a worker, it will not be created, and if a worker scoped fixture is used in a test, then it will remain instantiated for the duration of the worker. Overall this a very powerful feature that can help with expensive setup and teardown.
```

Comment:
This is super cool!

Reply:
This is also a super long comment ;)

Reply:
Addressed in [8a3c12c](https://github.com/iFixit/react-commerce/pull/1275/commits/8a3c12cd936cbf92b837962b9708fc5ad8d32fcc)